web/gui/dashboard_info.js

```diff
@@ -1310,8 +1310,8 @@ netdataDashboard.context = {
     },
 
     'system.clock_sync_state': {
-        info:'<p>The system clock synchronization state. '+
-        'It is strongly recommended having the clock in sync with reliable NTP servers. Otherwise, '+
+        info:'<p>The system clock synchronization state as provided by the ntp_adjtime() system call. '+
+        'This reflects the status of the system kernel regarding clock synchronization.'+ 
         'it leads to unpredictable problems. '+
```

Comment:
This needs rewording after the change.

Reply:
My first suggestion would be to change to
"an unsynchronized clock may be the result of synchronization issues by the NTP daemon, or the system timedatectl settings"

Reply:
I think we need to remove 

> This reflects the status of the system kernel regarding clock synchronization

because it is basically the same info as

> The system clock synchronization state as provided by the ntp_adjtime() call

---

> an unsynchronized clock may be the result of synchronization issues by the NTP daemon, or the system timedatectl settings.

Not a specialist, but I think it is "systemd-timesyncd" which is an NTP daemon too (client-side imlementation only).

Reply:
okay, so remove  "this reflects the status [...]" and the timedatectl reference?